.rubocop.yml

```diff
@@ -69,10 +69,10 @@ Lint/UselessAssignment:
     - '**/spec/**/*'
 
 # We could potentially enable the 2 below:
-Layout/IndentFirstHashElement:
+Layout/FirstHashElementIndentation:
```

Comment:
Looks like this PR includes changes from #171 ?

Reply:
Yeah it did. Not a problem anymore since #171 is merged :D